ci/jenkins/test.sh

```diff
@@ -316,7 +316,7 @@ function deliver_antrea_windows {
     fi
 
     cp -f build/yamls/*.yml $WORKDIR
-    docker save -o antrea-ubuntu.tar projects.registry.vmware.com/antrea/antrea-ubuntu:latest
+    docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
```

Comment:
The default registry is `DOCKER_REGISTRY=$(head -n1 "${WORKSPACE}/ci/docker-registry")`, there are quite a few scripts are using this default registry to build images. I feel you may need to double check it.

Reply:
could you clarify your concern? Tagged images, and in particular images used for testing (e.g., nginx) or tagged base images (e.g. antrea/openvswitch) should still be pulled from `projects.registry.vmware.com`. The Jenkins test scripts build the antrea image (`antrea/antrea-ubuntu:latest`) as part of the test (image is not pulled) and there is no reason to tag it with the registry (`projects.registry.vmware.com/antrea/antrea-ubuntu:latest`) any more.

Reply:
I mean there are some images' prefix is `${DOCKER_REGISTRY}` instead of `projects.registry.vmware.com` directly. E.g: https://github.com/antrea-io/antrea/blob/main/ci/jenkins/test-mc.sh#L268-L273.

Reply:
Thanks. As far as I can tell, only test-mc.sh needs to be updated.